ApplicationLayer/Revisal/Revisal_Close_Application.xaml

```diff
@@ -0,0 +1,88 @@
+<Activity mc:Ignorable="sap sap2010" x:Class="Revisal_Close_Application" mva:VisualBasic.Settings="{x:Null}" sap:VirtualizedContainerService.HintSize="1212,819" sap2010:WorkflowViewState.IdRef="Revisal_Close_Application_1" xmlns="http://schemas.microsoft.com/netfx/2009/xaml/activities" xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" xmlns:mva="clr-namespace:Microsoft.VisualBasic.Activities;assembly=System.Activities" xmlns:sap="http://schemas.microsoft.com/netfx/2009/xaml/activities/presentation" xmlns:sap2010="http://schemas.microsoft.com/netfx/2010/xaml/activities/presentation" xmlns:scg="clr-namespace:System.Collections.Generic;assembly=mscorlib" xmlns:sco="clr-namespace:System.Collections.ObjectModel;assembly=mscorlib" xmlns:ui="http://schemas.uipath.com/workflow/activities" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">

```

Comment:
I'm guessing the app sometimes needs to be closed twice? You could make a do while loop where you close the app and the check for the selector of it and if it's still there the loop continues

Reply:
Created a while loop that checks if the window still exists it will close it. Initially added two activities because when the app is opened, two almost identical windows appear and the second one wouldn't close upon closing the main one.